### ISSUE TITLE
Building with --without-ncurses still looks for curses.h

### ISSUE THREAD

**Issue by DimStar77** [adrianlopezroche/fdupes#143]
Even when fdupes 2.0.0 is configured using --without-ncurses, the build tries to include curses.h

config.h contains ```#undef HAVE_NCURSESW_CURSES_H``` - yet, AC_CONFIG_HEADER comments this out - and configure later on adds a ```#define NO_NCURSES``` - but that in turn does not match what fdupes.c cares for to decide on ```#include <ncursesw/curses.h>```

**Comment by DimStar77** [comment]
(otoh I don't grasp yet where the definition would be coming from for gcc still trying to load config.h)

**Comment by DimStar77** [comment]
oh.. doh!

```
#ifdef HAVE_NCURSESW_CURSES_H
  #include <ncursesw/curses.h>
#else
  #include <curses.h>
#endif
```

so it's only a decision if cursesw or not - but loading a variant of curses.h is unconditional

**Comment by adrianlopezroche** [comment]
Fixed by 8b10627adfe4f75b68a41b852ead8c269b16685b.
